draft-ietf-oauth-identity-chaining.md

```diff
@@ -274,6 +276,18 @@ Authorization servers MAY transcribe claims when either producing JWT authorizat
 
 The representation of transcribed claims and their format is not defined in this specification.
 
+### Use Cases
+
+Some use cases that demonstrate the need for claims description are as follows.
+
+Downscoping Permissions for API Access:
+
+A cloud-based development platform allows developers to access APIs across multiple trust domains. A developer in Domain A requests access to an API in Domain B but only needs limited permissions, such as "read-only" access. The authorization server in Domain A transcribes the claims in the JWT authorization grant to reflect the downscoped permissions, removing higher-privileged claims like "write" or "admin." This ensures that the access token issued by Domain B aligns with the developer's intended scope of access.
```

Comment:
How would Domain A's server know what permissions the user needs in domain B? Isn't this something that the authorization server in Domain B is more likely to enforce?